src/java.base/share/classes/java/lang/ScopedValue.java

```diff
@@ -248,7 +249,7 @@ Object find(ScopedValue<?> key) {
      * mapping bound to values. The following example runs an operation with {@code k1}
      * bound (or rebound) to {@code v1}, and {@code k2} bound (or rebound) to {@code v2}.
      * {@snippet lang=java :
-     *     // @link substring="where" target="#where(ScopedValue, Object)" :
+     *     // @link substring="runWhere" target="#runWhere(ScopedValue, Object)" :
```

Comment:
Is this correct?

Reply:
Good catch, might have been the victim of search and replace, it should continue to link to ScopedValue.where.